<launch>
  <node 
    pkg="livox_camera_calib"
    type="bag_to_pcd"
    name="bag_to_pcd"
    output="screen"
  />
   <param name="bag_file" type="string" value="/home/hzx/data/bag/"/>
   <param name="lidar_topic" type="string" value="livox/lidar"/>
   <param name="pcd_file" type="string" value="/home/hzx/data/bag/"/>
   <param name="data_num" type="int" value="6"/>
   <param name="is_custom_msg" type="bool" value="false"/>
</launch>